package.json

```diff
@@ -20,6 +24,11 @@
     "url": "https://github.com/FontoXML/fonto-benchmark-runner/issues"
   },
   "homepage": "https://github.com/FontoXML/fonto-benchmark-runner#readme",
+  "files": [
+    "bin/*",
+    "lib/*",
+    "lib/**/*"
```

Comment:
I think x/**/* already includes x/*

Reply:
I want to include all files in `lib` folder and all files in the folders in `lib` folder. Then `lib/*` is enough, right??

Reply:
I think lib/* is just all files directly in lib, but lib/**/* should be all files and folders in lib and in its subfolders. So lib/**/* includes lib/*.

Better test that to make sure though :)

Reply:
I tested it by using this `glob.sync(`${cwd}/performance/**/*.benchmark.ts`);` in `server.ts`. `x/**/*` pattern covers all files in x and in any deep subfolders of x. `x/*` pattern covers the files only in x. I think we should go with `x/**/*`

Reply:
Did you test `glob.sync(${cwd}/performance/**/*.benchmark.ts);` vs. `glob.sync(${cwd}/performance/*.benchmark.ts);` cause there I think it will indeed only files under performance due to the postfix check. If the glob doesn't include postfix than I think that `lib/*` should indeed be enough.